packages/core/frontend/sirius-components-core/src/onboarding/OnboardArea.tsx

```diff
@@ -69,7 +71,23 @@ const INITIAL_STATE = {
   representations: [],
 };
 
+const useOnboardAreaStyles = makeStyles(() => ({
+  grid: {
+    paddingTop: '48px',
+    paddingBottom: 2,
+    paddingRight: 2,
+    marginTop: 2,
+    marginLeft: '16px',
+    marginRight: '25px',
+    maxWidth: 'fit-content',
```

Comment:
- Use `theme.spacing(x)` instead of hardcoded values
- The max width feels odd, as usual this concern should be expressed in the parent component